CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ set(PRODUCT_BUILDDATE "1970-01-01" CACHE STRING "Product build date")
 set(PRODUCT_SOURCE_REVISION "0000000000000000000000000000000000000000" CACHE STRING "Product source SHA")
 set(PRODUCT_SOURCE_REPOSITORY "<unspecified>" CACHE STRING "Product source branch")
 set(PRODUCT_BITS "${BUILD_BITS}bit")
+set(GEODE_AUTHEXPIREDEXCEPTION_VERSION "1.15.0" CACHE STRING "First build containing AuthenticationExpiredException")
```

Comment:
Why is this a cache variable and why in the root project. This should just be a string literal in the javaobject project.

Reply:
See Martell's comment - this is a string variable because there may be different version(s) to test against, depending on which product is being built.

Reply:
Seems reasonable to put it in the cmake config file for the project it applies to.

Reply:
Removed.